crates/stc_ts_type_checker/tests/conformance/types/tuple/arityAndOrderCompatibility01.ts

```diff
@@ -33,3 +33,8 @@ var n3: [number, string] = z;
 var o1: [string, number] = x;
 var o2: [string, number] = y;
 var o3: [string, number] = y;
+
+x = y;
```

Comment:
Please revert this

Reply:
This test file is copied from the official tsc conformance test suite, so we should not modify it.